src/ch/imedias/rsccfx/model/Rscc.java

```diff
@@ -192,7 +195,7 @@ public String getKey() {
     return key.get();
   }
 
-  public void setKey(String key) {
+  private void setKey(String key) {
```

Comment:
Also thought about that, and did it now. BUt i set the setKey public again, beacuse the Supporter can set it from the view/presenter